SignUp.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>SignUp</title>
+  <link rel="stylesheet" href="login_SignUp.css">
+  <link rel="icon" href="img/logopanda.svg">
+</head>
+<body>
+  <main>
+    <a href="index.html">
+      <img class="main-logo" src="img/loginlogo.svg" alt="판다마켓로고">
+    </a>
+    <form>
+      <label class="input-label" for="username">
+        이메일
+      </label>
+      <input class="input-field" id="username" name="username" type="email" placeholder="codeit@email.com">
+
+      <label class="input-label" for="username">
+        닉네임
+      </label>
+      <input class="input-field" id="username" name="username" type="text" placeholder="판다">
+      
+      <label class="input-label" for="password" >
+        비밀번호
+      </label>
+      <input class="input-field input-password" id="password" name="password" type="password" placeholder="••••••••">
+     
+      <label class="input-label" for="password-repeat" >
+        비밀번호 확인
+      </label>
+      <input class="input-field input-password"  id="password-repeat" name="password-repeat" type="password" placeholder="asdffff">
+
+      <button class="sign-button">
+        회원가입
+      </button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
      <button type="submit" class="sign-button">
        회원가입
      </button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)